cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionCheckCommand.java

```diff
@@ -41,6 +42,10 @@
 
     static Logger logger = LoggerFactory.getLogger(ExtractionDiffCommand.class);
 
+    protected static final String PHAB_OVERRIDE_FLAG = "SKIP_I18N_CHECKS";
```

Comment:
knit: should the constant renamed to something like `SKIP_I18N_CHECKS_FLAG`? I'm not super clear about the "override" meaning. and then maybe `isOverrideActive()` is `areChecksSkipped()`?